quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -425,6 +477,20 @@ void produceDefaultPersistenceUnit(BuildProducer<PersistenceXmlDescriptorBuildIt
         producer.produce(new PersistenceXmlDescriptorBuildItem(descriptor));
     }
 
+    private void configurePersistenceUnitProperties(String datasourceName, ParsedPersistenceXmlDescriptor descriptor) {
```

Comment:
Will be used for other properties as mentioned here: https://github.com/keycloak/keycloak/compare/main...shawkins:keycloak:pu